src/main/java/entity/Editor.java

```diff
@@ -1,34 +0,0 @@
-package entity;
```

Comment:
by removing this are we just pushing logic downwards

Reply:
Editor served no purpose as it was replaced by EditorObj